about.html

```diff
@@ -0,0 +1,234 @@
+            <div class="text-center pt-5">
+                <h2 class="summit-heading pb-2">Techlife Innovative Global Summit 2023 Logo</h2>
+                <div class="line"></div>
+            </div>
+            <div class="row my-3 align-items-center justify-content-center g-1">
+                <div class="col-md-8 text-center">
+                    <p class="submit-p col-md-start">The logo of TechLife Global Summit 2023 was decided the logo competition from 8, June to 7, July.<br>
+                        After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+                        After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+                    </p>
+                </div>
+        
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class=" col-md-4 summit-logo p-2">
+                   <img id="summit" src="assets/images/logo/Tech.png" alt="techlife logo" class="img-fluid">
+                </div> 
+            </div>
+        </div>
+        <hr>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="past-summit">
+        <div class="container-lg">
+            <div class="text-center">
+                <h2 class="summit-heading pt-4 pb-2">See the Past TechLife Global Summits</h2>
+                <div class="line"></div>
+                <p class="past-p p-3">Take a look at the past two TechLife Global Summits which took place in Herat and in Kabul </p>
+            </div>
+            <div class="row photo-container justify-content-center pb-5 pt-2 g-1">
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container mb-4">
+                            <img src="assets/background/past-summit-1.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-3">
+                              <h2 class="img-header">2021</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2021</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-2.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-3">
+                              <h2 class="img-header">2022</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2022</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>   
```

Comment:
[Optional] 📷 Good job on implementing the images in your project! However, there is a small adjustment needed to align the text properly within the images. As per the design, the text should be centered within each image.Please update your code to center the text within the images, following the design guidelines. Keep up the great work! 👍🎉

| Your | Professional |
| --- | --- |
| ![image](https://github.com/MozamelJawad/Capstone-Project/assets/55840999/0001450d-9328-408f-9663-da7047cf470f)  | ![image](https://github.com/MozamelJawad/Capstone-Project/assets/55840999/5da6d053-7077-404d-8ddc-04f831777241) |